extensions/github-stars/package.json

```
@@ -0,0 +1,55 @@
+{
+  "name": "search-github-stars",
+  "title": "GitHub Stars",
+  "description": "Display and filter your recent GitHub stars",
+  "subtitle": "GitHub",
+  "icon": "icon.png",
+  "author": "mrmartineau",
```

```suggestion
  "author": "mrmartineau",
  "keywords": ["stars"],
```

This was it's still possible to search on "stars"